biz.aQute.bnd.exporters/bnd.bnd

```diff
@@ -9,7 +9,8 @@ Conditional-Package:                \
     aQute.libg.*;-split-package:=first
 
 -buildpath: \
-    aQute.libg;version=project, \
-	biz.aQute.bndlib;version=latest, \
-	osgi.core;version=@6, \
-	osgi.cmpn;version=@6
+	aQute.libg;version=project,\
+	biz.aQute.bndlib;version=latest,\
+	osgi.core;version=@6,\
+	osgi.cmpn;version=@6,\
+	org.apache.servicemix.bundles.junit;version=4.12
```

Comment:
You should not add junit to the `-buildpath`. Add `-testpath: ${junit}` to the bnd file to get the proper junit jars on the test path.